release/air_tests/air_benchmarks/workloads/xgboost_benchmark.py

```diff
@@ -122,23 +122,32 @@ def main(args):
     with open(test_output_json, "wt") as f:
         json.dump(result, f)
 
-    if training_time > _TRAINING_TIME_THRESHOLD:
-        raise RuntimeError(
-            f"Training on XGBoost is taking {training_time} seconds, "
-            f"which is longer than expected ({_TRAINING_TIME_THRESHOLD} seconds)."
-        )
+    if not args.no_timeout:
+        if training_time > _TRAINING_TIME_THRESHOLD:
+            raise RuntimeError(
+                f"Training on XGBoost is taking {training_time} seconds, "
+                f"which is longer than expected ({_TRAINING_TIME_THRESHOLD} seconds)."
+            )
 
-    if prediction_time > _PREDICTION_TIME_THRESHOLD:
-        raise RuntimeError(
-            f"Batch prediction on XGBoost is taking {prediction_time} seconds, "
-            f"which is longer than expected ({_PREDICTION_TIME_THRESHOLD} seconds)."
-        )
+        if prediction_time > _PREDICTION_TIME_THRESHOLD:
+            raise RuntimeError(
+                f"Batch prediction on XGBoost is taking {prediction_time} seconds, "
+                f"which is longer than expected ({_PREDICTION_TIME_THRESHOLD} seconds)."
+            )
 
 
 if __name__ == "__main__":
     import argparse
 
     parser = argparse.ArgumentParser()
     parser.add_argument("--size", type=str, choices=["10G", "100G"], default="100G")
+    # Add a flag for disabling the timeout error.
+    # Use case: running the benchmark as a documented example, in infra settings
+    # different from the formal benchmark's EC2 setup.
+    parser.add_argument(
+        "--no-timeout",
```

Comment:
Instead of this, can we call it `--disable-check`?

Reply:
sgtm

Reply:
fixed